_episodes/034-rust-1.52-1.53.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "What's New in Rust 1.52 and 1.53"
+date: 2021-07-13T12:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e034-rust-1.52-1.53.mp3
+duration: "1:07:08"
+length: "48336666"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Jon and Ben discuss the highlights of the 1.52 and 1.53 releases of Rust.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+#### [@01:31] - [Rust 1.52](https://blog.rust-lang.org/2021/05/06/Rust-1.52.0.html)
+
+ - [@01:31] - [Stabilized APIs](https://blog.rust-lang.org/2021/05/06/Rust-1.52.0.html#stabilized-apis)
+ - [@04:28] - [All integer division and remainder APIs made const](https://github.com/rust-lang/rust/pull/80962)
+ - [@07:45] - [Rust 1.52.1 and incremental compilation](https://blog.rust-lang.org/2021/05/10/Rust-1.52.1.html)
+ - [@11:30] - [LLVM 12](https://github.com/rust-lang/rust/pull/81451)
+     - [Disable "mutable noalias"](https://github.com/rust-lang/rust/issues/84958)
+     - [Bringing Stack Clash Protection to Clang/x86, the Open Source Way](https://blog.llvm.org/posts/2021-01-05-stack-clash-protection/)
+ - [@16:15] - [`unsafe_op_in_unsafe_fn` lint](https://github.com/rust-lang/rust/pull/79208)
+
+#### [@23:03] - [Rust 1.53](https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html)
+
+ - [@23:03] - [`IntoIterator` for arrays](https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#intoiterator-for-arrays)
+ - [@26:45] - [Unicode identifiers](https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#unicode-identifiers)
+ - [@29:37] - [Or patterns](https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#or-patterns)
+ - [@31:05] - [Stabilized APIs](https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#stabilized-apis)
```

Comment:
Should we also link to the [`BITS` discussion](https://github.com/rust-lang/rust/issues/81654)?